Source/WebCore/css/parser/CSSParserContext.cpp

```diff
@@ -149,7 +150,8 @@ void add(Hasher& hasher, const CSSParserContext& context)
         | context.cssProgressFunctionEnabled                << 25
         | context.cssMediaProgressFunctionEnabled           << 26
         | context.cssContainerProgressFunctionEnabled       << 27
-        | (uint32_t)context.mode                            << 28; // This is multiple bits, so keep it last.
+        | context.cssRandomFunctionEnabled                  << 28
+        | (uint32_t)context.mode                            << 29; // This is multiple bits, so keep it last.
```

Comment:
How many bits? We are getting perilously 32 here.

Reply:
It needs 3 bits, so just barely still works. Next addition will kill it.

Reply:
Stuff like the Enabled bits probably doesn't even need to be part of the hash. Who cares if we get some collisions when someone flips the setting?